packages/compartment-mapper/src/node-powers.js

```diff
@@ -1,5 +1,7 @@
 // @ts-check
 
+import './types.js';
+
 /**
  * @param {typeof import('fs')} fs
  * @returns {ReadPowers}
```

Comment:
To avoid the `import './types.js';` above:

```suggestion
 * @returns {import('./types.js').ReadPowers}
```